src/Components/Parts/ChatInput.vue

```diff
@@ -43,11 +43,10 @@
     </div>
 </template>
 
-<style lang="sass" scoped>
+<style lang="sass">
 @import '@/Style/Mixins'
 
 .bottomchat
-    position: fixed
```

Comment:
why position fixed?

Reply:
we needed to remove it to make it dynamic